app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class RelationshipsController < ApplicationController
+  def create
+    @user = User.find(params[:followed_id])
+    current_user.follow(@user)
+    get_response(@user)
+  end
+
+  def destroy
+    @user = Relationship.find(params[:id]).followed
+    current_user.unfollow(@user)
+    get_response(@user)
+  end
+
+  def get_response(user)
```

Comment:
I'd change the name of this function, maybe something like `respond_with_format` or something like that